feedthefox/settings.py

```diff
@@ -9,6 +9,11 @@
 """
 
 import os
+try:
+    from urllib.parse import urljoin
```

Comment:
Indeed, I was using at first but when I removed it I forgot this reference. Nice catch